runtime/dancebox/src/xcm_config.rs

```diff
@@ -245,14 +249,20 @@ impl pallet_xcm::Config for Runtime {
     type AdminOrigin = EnsureRoot<AccountId>;
 }
 
+pub type PriceForSiblingParachainDelivery =
+    ExponentialPrice<SelfReserve, BaseDeliveryFee, TransactionByteFee, XcmpQueue>;
+
+pub type PriceForParentDelivery =
+    ExponentialPrice<FeeAssetId, BaseDeliveryFee, TransactionByteFee, ParachainSystem>;
```

Comment:
Why do we need the conversion to `FeeAssetId` type? Couldn't we use the same `SelfReserve` as in `PriceForSiblingParachainDelivery`?

Reply:
yep I think so